examples/SC_noise_capture.py

```diff
@@ -116,7 +116,7 @@ def collect(config_dict):
                 .setaxis("t", time_axis)
                 .set_units("t", "s")
                 .setaxis("nScans", np.zeros(config_dict["nScans"]))
-                .name("signal")
+                .name("noise")
```

Comment:
can we call it `noisecapture` or something like that? calling it just noise is weird to me